openmetadata-ui/src/main/resources/ui/src/pages/TagPage/TagPage.tsx

```diff
@@ -0,0 +1,579 @@
+import { getCountBadge, getEntityDeleteMessage } from '../../utils/CommonUtils';
+import { getEntityName } from '../../utils/EntityUtils';
+import { checkPermission } from '../../utils/PermissionsUtils';
+import {
+  getClassificationDetailsPath,
+  getClassificationTagPath,
+} from '../../utils/RouterUtils';
+import {
+  escapeESReservedCharacters,
+  getEncodedFqn,
+} from '../../utils/StringsUtils';
+import { showErrorToast, showSuccessToast } from '../../utils/ToastUtils';
+import { TagTabs } from './TagPage.inteface';
+
+const TagPage = () => {
+  const { t } = useTranslation();
+  const { fqn: tagFqn } = useFqn();
+  const history = useHistory();
+  const { tab: activeTab = TagTabs.OVERVIEW } = useParams<{ tab?: string }>();
+  const { permissions } = usePermissionProvider();
+  const [isLoading, setIsLoading] = useState(false);
+  const [tagItem, setTagItem] = useState<Tag>();
+  const [isDescriptionEditable, setIsDescriptionEditable] =
+    useState<boolean>(false);
+  const [isNameEditing, setIsNameEditing] = useState<boolean>(false);
+  const [isStyleEditing, setIsStyleEditing] = useState(false);
+  const [isDelete, setIsDelete] = useState<boolean>(false);
+  const [showActions, setShowActions] = useState(false);
+  const [assetCount, setAssetCount] = useState<number>(0);
+  const assetTabRef = useRef<AssetsTabRef>(null);
+  const [previewAsset, setPreviewAsset] =
+    useState<EntityDetailsObjectInterface>();
+  const resourceType = ResourceEntity.TAG;
+  const breadcrumb: TitleBreadcrumbProps['titleLinks'] = useMemo(() => {
+    return tagItem
+      ? [
+          {
+            name: 'Classifications',
+            url: ROUTES.TAGS,
+            activeTitle: false,
+          },
+          {
+            name: tagItem.classification?.name ?? '',
+            url: tagItem.classification?.fullyQualifiedName
+              ? getClassificationDetailsPath(
+                  tagItem.classification.fullyQualifiedName
+                )
+              : '',
+            activeTitle: false,
+          },
+        ]
+      : [];
+  }, [tagItem]);
+
+  const handleAssetClick = useCallback(
+    (asset?: EntityDetailsObjectInterface) => {
+      setPreviewAsset(asset);
+    },
+    []
+  );
+
+  const hasEditPermission = useMemo(() => !tagItem?.disabled, [tagItem]);
+
+  const onDescriptionUpdate = async (updatedHTML?: string) => {
+    if (tagItem) {
+      if (tagItem.description !== updatedHTML) {
+        const updatedTableDetails = {
+          ...tagItem,
+          description: updatedHTML,
+        };
+        const jsonPatch = compare(tagItem, updatedTableDetails);
+        try {
+          const response = await patchTag(tagItem.id ?? '', jsonPatch);
+
+          setTagItem(response);
+
+          if (tagItem?.name !== updatedTableDetails.name) {
+            history.push(
+              getClassificationTagPath(response.fullyQualifiedName ?? '')
+            );
+          }
```

Comment:
this is not required, you are only updating description.

Reply:
Updated